.github/workflows/html-build.yml

```
@@ -0,0 +1,34 @@
+name: Rebuild all with asciidoctor
+
+on:
+  # Runs on pushes targeting the main branch.
+  push:
+    branches: ["main"]
+  # Runs on pull requests, uses the action's description from the target branch.
+  pull_request_target:
+    types: [opened,synchronize]
+
+  workflow_dispatch:
+
+jobs:
+  # Build job
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt -y install dos2unix
+          sudo apt -y install asciidoctor
```

Might be good to use the docker image (as suggested in the build docs and as used in Vulkan-Docs)

```suggestion
    # We now refer to the container by its SHA instead of the name, to prevent
    # caching problems when updating the image.
    # container: khronosgroup/docker-images:asciidoctor-spec.20240726
    container: khronosgroup/docker-images@sha256:089687083ceb36483a3917389e4278718ab19c594099634f5dd80e22540c960f
    steps:
      - name: Checkout
        uses: actions/checkout@v3
```